answers/AbstractClassesAndMethods/0b8d656f-d71c-4787-9ba2-a16eaeeec79e/Student.php

```diff
@@ -0,0 +1,26 @@
+<?php
+require_once(__DIR__ . '/Person.php');
+class Student extends Person
+{
+    private $course;
+    	
+    public function __construct($name, $age, $course)
+    {
+        parent::__construct($name, $age);
+        $this->course = $course;
+    }
+    
+    public function getCourse()
+    {
+        return $this->course;
+    }
+
+    public function display()
+    {
+        echo parent::getName() . " is taking up " . $this->course . ".";
```

Comment:
Why use `parent::getname()` in here?

Reply:
This is to show that `parent` can be used also to access the methods of the parent class in the Child class.

Reply:
with your example, you can just do `$this->getName()` and it will have the same effect, since your subclass has no `getName` method. 

If we want to show the student that `parent` can be used to call the parent method, let's do this by overwriting the method `getName` e.g.

```
// In Child class
public function getName()
{
    return strtoupper(parent::getName() ) 
}

public function display()
{
   echo $this->getName() . " is taking up " . $this->course . ".";
}
```